xbmc/addons/AddonDatabase.cpp

```diff
@@ -862,7 +868,8 @@ bool CAddonDatabase::Search(const std::string& search, VECADDONS& addons)
   return false;
 }
 
-bool CAddonDatabase::DisableAddon(const std::string &addonID, bool disable /* = true */)
+bool CAddonDatabase::DisableAddon(const std::string& addonID,
+                                  const AddonDisabledReason& disabledReason)
```

Comment:
think about a enum is a "const ...&` not needed. 
@ksooo brought me to it.

Reply:
It's not required but I prefer it has it indicates this value will not be modified. Plus if it becomes a class later on the signature does not change.